tests/system_tests_multi_tenancy.py

```diff
@@ -683,12 +704,8 @@ def __init__(self, first_host, second_host, first_address, second_address, dynam
         self.n_settled = 0
 
     def timeout(self):
-        self.error = "Timeout Expired: n_sent=%d n_rcvd=%d n_settled=%d" % (self.n_sent, self.n_rcvd, self.n_settled)
-        self.first_conn.close()
-        self.second_conn.close()
-        self.lookup_conn.close()
-        if self.poll_timer:
-            self.poll_timer.cancel()
+        self.fail("Timeout Expired: n_sent=%d n_rcvd=%d n_settled=%d" %
+                  (self.n_sent, self.n_rcvd, self.n_settled))
```

Comment:
All the other tests scrupulously avoid throwing in the Proton handlers. I presume there was/is a good reason for it. (Other tests have the `error` field, and always do `self.assertIsNone(test.error)` at the end). So, is throwing actually alright?

Reply:
@kgiusti Actually, you don't even have `self.fail` available here! This is instance of `proton.handlers.MessagingHandler`, not `unittest.TestCase`. (At least, If I look right).

Reply:
Yeah - not the best choice of method name, but it's local to the LinkRouteTest class itself.  I've changed it from "fail" to "done" so it won't be as confusing.